src/Symfony/Component/Clock/MonotonicClock.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Clock;
+
+/**
+ * A monotonic clock suitable for performance profiling.
+ *
+ * @author Nicolas Grekas <p@tchwork.com>
+ */
+final class MonotonicClock implements SleepableClockInterface
+{
+    private int $sOffset;
+    private int $µsOffset;
```

Comment:
As someone who doesn't have Greek letters on his keyboard, I'd like to propose sticking to ASCII characters for variable names. 😓

Reply:
granted :)